Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -134,7 +134,7 @@ class Tab: NSObject {
   var bars = [SnackBar]()
   var favicon: Favicon
   var lastExecutedTime: Timestamp?
-  var sessionData: SessionData?
+  var sessionData: (title: String, interactionState: Data)?
```

Comment:
Why do we need title in sessionData tuple? We have a title property in the Tab already ?

Reply:
This is not the same thing. When restoring a tab, we restore the original title of that tab. The tab's title can currently be anything BEFORE session restore. A tab can be restored from recently closed or anywhere else. So we need to restore its title that was with its data.